### ISSUE TITLE
Pangolin_101 fails to compile

### ISSUE THREAD

**Issue by AcuarioCat** [philbowles/PangolinMQTT#38]
Pangolin_101.ino: 55:10: error: 'VARK_NO_SERVER_DETAILS' was not declared in this scope
Error compiling project sources

The declaration is commented out in AardvarkTCP.h so compilation fails.

**Comment by Josifjoey** [comment]
Same issue here, I've been working on it for the past couple of days, is there any way around this?

**Comment by luebbe** [comment]
Yes, use https://github.com/bertmelis/espMqttClient. It works and you get support there.

**Comment by HamzaHajeir** [comment]
You can check [H4AsyncMQTT](https://github.com/hamzahajeir/h4asyncmqtt) which is based on Pangolin in essence.
